src/modules/songs/components/ConfigureSongForSet/ConfigureSongForSet.tsx

```
@@ -358,15 +357,21 @@ export const ConfigureSongForSet: React.FC<ConfigureSongForSetProps> = ({
                 <div className="flex gap-2 text-slate-500">
                   <div className="flex items-center gap-1">
                     <Heart />
-                    <Text style={isDesktop ? "body-small" : "small"}>
+                    <Text
+                      style={textSize === "text-base" ? "body-small" : "small"}
+                    >
                       Preferred key: {formatSongKey(selectedSong.preferredKey!)}
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)